defaults/main.yml

```diff
@@ -73,65 +70,65 @@ role_parameters: {}
 # Addresses configuration (array of hashes).
-metrics_qdr_addresses: [{'prefix': 'collectd', 'distribution': 'multicast'}, {'prefix': 'ceilometer/metering.sample', 'distribution': 'multicast'}, {'prefix': 'ceilometer/event.sample', 'distribution': 'multicast'}]
+qdr_addresses: [{'prefix': 'collectd', 'distribution': 'multicast'}, {'prefix': 'ceilometer/metering.sample', 'distribution': 'multicast'}, {'prefix': 'ceilometer/event.sample', 'distribution': 'multicast'}]
 
 # Corresponds to MetricsQdrUsername in THT
 # Username which should be used to authenticate to the deployed qdrouterd.
-metrics_qdr_username: guest
+qdr_username: guest
 
 # Corresponds to MetricsQdrPort in THT
 # Service name or port number on which the qdrouterd will accept connections.
-metrics_qdr_port: 5666
+qdr_listener_port: 5666
+
+qdr_log_output: /var/log/qdrouterd/metrics_qdr.log
 
 # Corresponds to MetricsQdrLoggingSource in THT
 metrics_qdr_logging_source: {'startmsg.regex': '^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}(.[0-9]+ \\+[0-9]+)? [A-Z]+ \\([a-z]+\\) ', 'tag': 'openstack.nova.consoleauth', 'file': '/var/log/containers/qdrouterd/metrics_qdr.log'}
 
-# ContainerMetricsQdrImage has no default set in THT, so container_metrics_qdr_image
-# will not be set, but my be passed to this role anyway.
-# container_metrics_qdr_image : <undefined>
-
 # Corresponds to ServiceData in THT
 # Dictionary packing service data
-service_data: {}
+
+# ***** Don't need?
+#tripleo_service_data: {}
 
 # Corresponds to MetricsQdrSSLProfiles in THT
 # SSL Profiles for the connectors (array of hashes).
-metrics_qdr_sslprofiles: []
+qdr_ssl_profiles: []
 
 # Corresponds to EndpointMap in THT
 # Mapping of service endpoint -> protocol. Typically set via parameter_defaults
 # in the resource registry.
-endpoint_map: {}
+# ***** Don't need?
+tripleo_endpoint_map: {}
 
 # Corresponds to MetricsQdrSslCertFile in THT
 # Path to SSL certificate file for listener.
-metrics_qdr_ssl_cert_file: 
-
-# ContainerMetricsQdrConfigImage has no default set in THT, so container_metrics_qdr_config_image
-# will not be set, but my be passed to this role anyway.
-# container_metrics_qdr_config_image : <undefined>
+qdr_listener_ssl_cert_file: /etc/pki/tls/certs/metrics_qdr.crt
 
 # Corresponds to MetricsQdrSslCertDb in THT
 # Path to SSL certificate db for listener.
-metrics_qdr_ssl_cert_db: 
+qdr_listener_ssl_cert_db: /etc/pki/tls/certs/ca-bundle.crt
 
 # Corresponds to MetricsQdrSaslMechanisms in THT
 # List of accepted SASL auth mechanisms for listener in format of comma separated
 # list.
-metrics_qdr_sasl_mechanisms: ANONYMOUS
+qdr_listener_sasl_mech: ANONYMOUS
 
 # Corresponds to MetricsQdrSslPassword in THT
 # SSL password to be supplied for listener.
-metrics_qdr_ssl_password: 
+qdr_listener_ssl_password:
 
 # Corresponds to MetricsQdrSslKeyFile in THT
 # Path to SSL private key file for listener.
-metrics_qdr_ssl_key_file: 
+qdr_listener_ssl_key_file: /etc/pki/tls/certs/metrics_qdr.key
```

Comment:
For the var names here, with collectd, I converted the var names in THT to snake_case, so it would be easier to convert the parameter names from THT to their ansible equivalents.

Reply:
Ah, like this, you mean: https://review.opendev.org/c/openstack/tripleo-heat-templates/+/737100/7/deployment/metrics/collectd-container-ansible.yaml#410  I'll consider that and see if it will add any clarity of make life easier somehow